stock_scanner/README.rst

```diff
@@ -0,0 +1,187 @@
+====================================================
```

Comment:
Hi @sylvain-garancher Thanks for your contribution
Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst